src/pages/news/news-page/news.js

```diff
@@ -33,7 +33,7 @@ const NewsPage = ({ getNews, list }) => {
   );
 };
 
-const mapStateToProps = ({ News: { list } }) => ({
+const mapStateToProps = ({ news: { list } }) => ({
```

Comment:
It should be uppercased, rollback changes

Reply:
Done